src/components/DepositWidget/index.tsx

```diff
@@ -60,9 +68,28 @@ const LinkWrapper = styled.div`
   }
 `
 
+const txOptionalParams: TxOptionalParams = {
+  onSentTransaction: (receipt: Receipt): void => {
+    const notification = <TxNotification txHash={receipt.transactionHash} />
+    if (notification) {
+      toast.info(notification)
+    } else {
+      console.error(`Failed to get notification for tx ${receipt.transactionHash}`)
+    }
+  },
+}
+
 const DepositWidget: React.FC = () => {
-  const { balances, error } = useTokenBalances()
+  const { userAddress } = useWalletConnection()
+  const { balances, setBalances, error } = useTokenBalances()
+
   const contractAddress = depositApi.getContractAddress()
+  const mounted = useRef(true)
+  useEffect(() => {
+    return function cleanUp(): void {
+      mounted.current = false
+    }
+  }, [])
 
   if (balances === undefined) {
     // Loading: Do not show the widget
```

Comment:
To render nothing, better return `null`, not a Fragment

Reply:
weird cause in my code I have:

![image](https://user-images.githubusercontent.com/2352112/67779162-88c81b80-fa64-11e9-9a20-728a69b15469.png)

Reply:
I updated it in another PR :)